### ISSUE TITLE
Installation in Docker on Mac M1 Processor

### ISSUE THREAD

**Issue by bechtold** [google/or-tools#2516]
Hello,
I'm getting an "ERROR: Could not find a version that satisfies the requirement ortools" all the time when trying to install ortools via pip inside a docker.

**What version of OR-Tools and what language are you using?**
Version: latest
Language: Python

**What operating system (Linux, Windows, ...) and version?**
I tried different Linux images in docker desktop for mac on an m1 processor. Docker needs rosetta 2 to run.
I tried the following images:
FROM python:3-buster
FROM python:3.9.2-buster
FROM python:3.8-buster
FROM ubuntu:18.04
FROM ubuntu:20.04

**What did you do?**
Start a docker container and run `pip install ortools` or `pip install -r requirements.txt`

**What did you expect to see**
A working installation of ortools.

**What did you see instead?**
ERROR: Could not find a version that satisfies the requirement ortools (from versions: none)
ERROR: No matching distribution found for ortools

**Anything else we should know about your project / environment**
I guess this has to do with the virtualization stuff. Docker Desktop needs to run in rosetta because it does not natively run on silicon chips. I'm not an expert on this, so it just took me three hours trying and resolve this bug and I had to give up. Switching to a Linux machine and installing ortools in docker there worked like a charm. Also, ortools run on a native installation on M1, that's why I initially didn't see tho problem with the M1. So I'm posting this mainly to let others know this problem currently exists, so they don't waste the same amount of time like I just did :-)

**Comment by lperron** [comment]
Duplicate

**Comment by Mizux** [comment]
to be precise duplicate of #2332

**Comment by bechtold** [comment]
Thx for the link, I was actually wondering why I didn't find it while searching for my issue.

I saw issue #2332, but that is about natively running or-tools on an M1 processor.
It does run perfectly on my M1, it just doesn't run in a Docker on an M1. So I believe that my problem is different from #2332.

But I solved my problem by moving to another computer and creating my Dockerfile there. So I'm happy with leaving this issue closed.
